src/MCPServer/lib/assets/workflow.json

```diff
@@ -0,0 +1,14741 @@
+{
+    "chains": {
```

Comment:
Hi @sevein, you might see me 'unresolve' a handful of conversations in this PR. It is hard to see what has and hasn't been addressed and why if the conversation is resolved without comment. 

I have been trying to read up on code review to improve my feedback, but also how I respond to changes to myself. I found this a useful blog, #7 here helps to explain a little more about why this can be a difficult for the reviewer: https://medium.freecodecamp.org/unlearning-toxic-behaviors-in-a-code-review-culture-b7c295452a3c

Reply:
Sorry, I shouldn't have done that.

> Do we need to identify our own schema version here so we know which workflow.json validates against which schema? I can't see explicit provisions for this in the JSON schema specification, and examples don't show $schema used outside of a schema itself... so maybe not?

This is captured in https://gist.github.com/sevein/63e751acd37bde5590d680344ad8b3d5 as a future improvement (`Version the document`).